docs/index.md

```diff
@@ -35,12 +74,39 @@ By default, the provider will use the Mondoo CLI configuration file to authentic
 configuration file is located at `~/.config/mondoo/mondoo.yml` on Linux and macOS, and `%HomePath%\mondoo\mondoo.yml`
 on Windows.
 
-You can alternatively use the `credentials` field, or any of the following environment variables ordered by precedence.
+You can alternatively, use the following environment variables ordered by precedence:
```

Comment:
```suggestion
You can alternatively use the following environment variables ordered by precedence:
```

Reply:
Looks like this stuff needs to get fixed elsewhere so the generated content includes the fixes